siriuspy/siriuspy/clientarch/pvarch.py

```diff
@@ -7,15 +7,20 @@
 from mathphys.functions import save_pickle as _save_pickle, \
     load_pickle as _load_pickle
 
+from .. import envars as _envars
+
 from . import exceptions as _exceptions
 from .client import ClientArchiver as _ClientArchiver
 from .time import Time as _Time, get_time_intervals as _get_time_intervals
 
 
 class _Base:
 
-    def __init__(self, connector=None):
+    DEF_PARALLEL_QUERY_BIN_INTERVAL = 12*60*60  # 12h
+
+    def __init__(self, connector=None, offline_server=False):
```

Comment:
It would be nice to have a mechanism to allow changing the server "in flight", don't you think?

Reply:
but there is. you can use
```Python3

from siriuspy import clientarch as carch

pvname = 'Si-Glob:AP-CurrInfo:Current-Mon'
pvd = carch.PVData(pvdata)
...

# change to offline archiver
pvd.connector.server_url = carch.SERVER_OFFLINE_URL)
```
but I also added helper `switch_to` methods do clientarch class hierarchy to make it easier. take a new look.